src/main/java/it/gov/pagopa/pu/bff/service/organization/BrokerServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package it.gov.pagopa.pu.bff.service.organization;
+
+import it.gov.pagopa.pu.bff.config.DefaultConfigFe;
+import it.gov.pagopa.pu.bff.connector.OrganizationClientImpl;
+import it.gov.pagopa.pu.bff.security.SecurityUtils;
+import it.gov.pagopa.pu.p4pa_organization.model.generated.EntityModelBroker;
+import it.gov.pagopa.pu.p4pa_organization.model.generated.EntityModelOrganization;
+import it.gov.pagopa.pu.p4pa_organization.model.generated.PersonalisationFe;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserInfo;
+import it.gov.pagopa.pu.p4paauth.model.generated.UserOrganizationRoles;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BrokerServiceImpl implements BrokerService{
+
+  private OrganizationClientImpl organizationClient;
+
+  private DefaultConfigFe defaultConfigFe;
+
+  public BrokerServiceImpl(OrganizationClientImpl organizationClient,
+      DefaultConfigFe defaultConfigFe){
+    this.organizationClient = organizationClient;
+    this.defaultConfigFe = defaultConfigFe;
+  }
+
+  @Override
+  public PersonalisationFe getBrokerConfig(){
+    UserInfo user = SecurityUtils.getLoggedUser();
```

Comment:
SecurityUtils should be called just on the Controller! Where to find it is an information that belong just to the presentation layer, not the business logic!
Business logic should receive it as a parameter

Reply:
This has not been resolved